ms/MeasurementSets/MSPointingColumns.cc

```diff
@@ -129,11 +129,11 @@ MDirection ROMSPointingColumns::sourceOffsetMeas(Int row,
     for (Int i=start; i<end; i++) {
       if (antennaId()(i)==antenna) {
 	Double halfInt=0.0;  
-	if(interval()(i)==0){
+	if(interval()(i)==0.0){
 	  Int counter=0;
 	  Int adder=1;
 	  
-	  while(time()(i+counter)==time()(i)){
+	  while(!( (time()(i+counter)!=time()(i))&& (antennaId()(i+counter) == antennaId()(i)))){
```

Comment:
In theory you are right... in practice an antenna will appear more than 
once in a pointing table.
Fixed it for that...just in  case

Thanks
Kumar

On 06/10/2016 04:51 AM, Julian Taylor wrote:
>
> In ms/MeasurementSets/MSPointingColumns.cc 
> <https://github.com/casacore/casacore/pull/437#discussion_r66595382>:
>
> >  	  Int counter=0;
> >  	  Int adder=1;
> >  	
> > -	  while(time()(i+counter)==time()(i)){
> > +	  while(!( (time()(i+counter)!=time()(i))&& (antennaId()(i+counter) == antennaId()(i)))){
>
> |antennaId()(i)| can just be |antenna|
>
> it will endless loop (or crash?) when the antenna id does not appear 
> anymore (in addition to all times being equal), though I assume that 
> does not happen?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/casacore/casacore/pull/437/files/9d089c9a5dee7fcb378b9cd905381fec14a6c346#r66595382>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AMg2aurdJ2kMj--d1rhXfVN5E1Cnrsy8ks5qKUHGgaJpZM4IyW0g>.Web 
> Bug from 
> https://github.com/notifications/beacon/AMg2an6wMcFJ70P9eP1O5L5SuFx1TdH8ks5qKUHGgaJpZM4IyW0g.gif
>